code/_global_vars/sound.dm

```diff
@@ -108,3 +108,8 @@ GLOBAL_LIST_INIT(glasscrack_sound,\
 		'sound/effects/glass_crack2.ogg',\
 		'sound/effects/glass_crack3.ogg',\
 		'sound/effects/glass_crack4.ogg'))
+
+GLOBAL_LIST_INIT(light_strike_sound,\
```

Comment:
`GLOB` is deprecated, you can just do `var/list/light_strike_sound` and then refer to `global.light_strike_sound` instead of `GLOB`.